docs/win32.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Windows Support
+---
+
+## Introduction
+
+The Windows MSI package is built automatically in continuous integration.
+Only some plugins are included, any plugins that use Linux-specific features like `udev`, `efivarfs` or `devfs` are disabled.
+However, some USB devices updatable in Linux are also updatable in Microsoft Windows and macOS too.
+
+## Installation
+
+First, install the `.msi` package from the [release announcements](https://github.com/fwupd/fwupd/releases/), or any of the CI artifacts such as `dist/setup/fwupd-1.9.10-setup-x86_64.msi`.
+
+Then click **More info** and **Run anyway** buttons:
+
+![run anyway](win32-run-anyway.png)
+
+If not already an administrator, click "yes" on the User Account Control (UAC) dialog:
+
+![uac](win32-uac.png)
+
+There are currently no start menu or desktop installed, although this is something we want to add in the future.
+In the meantime, use the start menu to open a *Command Prompt* and then make sure to click the **Run as an administrator** link:
```

Comment:
You don't actually need admin to interact with USB on Windows.  I _think_ this can instead be open powershell (and no need to run as admin).

Reply:
Agreed -- there's a harmless warning about failing to create a registry key (to set up the logging domain?) but it seems to work well enough without. I've reworded things a bit.